101-azure-api-for-fhir/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
remove this file if it's not available in Azure Gov

Reply:
@bmoore-msft done